zk/stages/stage_witness.go

```diff
@@ -182,9 +183,17 @@ func SpawnStageWitness(
 			return fmt.Errorf("ExecuteBlockEphemerallyZk: %w", err)
 		}
 
+		forcedInfoTreeUpdate, err := witness.CheckForForcedInfoTreeUpdate(memHermezDb, block.NumberU64())
```

Comment:
maybe we can leave a comment about what this exactly is? maybe even with GH issue link.

Reply:
like, today I do have all the context, but I can imagine a few months from now, it will be puzzling

Reply:
Ah yeah, I have one in the other file, will copy over